### ISSUE TITLE
Contrast ratio issues

### ISSUE THREAD

**Issue by axamei** [pulibrary/pul_library_drupal#1229]
Contrast ratio errors based on WCAG2AA test.

Front page: 
- main search block's search button. Recommendation: change background to #c05800.

All search page:
- blacklight search results availability of on-site access. Recommendation: change text color to #010100.
